src/effects/Reverb.cpp

```diff
@@ -233,6 +237,26 @@ struct EffectReverb::Instance
    size_t RealtimeProcess(size_t group, EffectSettings& settings,
       const float* const* inbuf, float* const* outbuf, size_t numSamples) override
    {
+
+      const auto& incomingSettings = GetSettings(settings);
+      if ( !(incomingSettings == mLastAppliedSettings) )
+      {
+         for (auto& slave : mSlaves)
+         {
+            for (unsigned int i = 0; i < slave.mState.mNumChans; i++)
+            {
+               auto& reverbCore = slave.mState.mP[i].reverb;
+               const auto& is = incomingSettings;
+               reverb_init(&reverbCore, mLastSampleRate,
+                           is.mWetGain, is.mRoomSize, is.mReverberance, is.mHfDamping,
+                           is.mPreDelay, is.mStereoWidth, is.mToneLow, is.mToneHigh   );
+            }
+         }         
+
+         mLastAppliedSettings = incomingSettings;
+      }
```

Comment:
The block of code above might instead be inserted into a new override of RealtimeProcessStart, to be more consistent with how other effects receive notifications of changes of settings, and also later to allow this to be rewritten to use EffectMessage instead.

For now it is not a blocker to merge this.